R/build-articles.R

```diff
@@ -244,7 +244,8 @@ build_article <- function(name,
     pagetitle = front$title,
     opengraph = list(description = front$description %||% pkg$package),
     source = repo_source(pkg, path_rel(input, pkg$src_path)),
-    filename = path_file(input)
+    filename = path_file(input),
+    output_file = output_file
```

Comment:
at the moment when rendering the Mustache template for articles there's no information on the actual output file, which is needed to activate the navbar.